### DISCUSSION TITLE
大家好

### DISCUSSION THREAD

**Discussion by czy3048** [share121/inter-knot#2505] [其他]
第一次进入绳网，希望和大家好好相处
![1723729402231](https://github.com/user-attachments/assets/6219ed29-558d-4c6d-91db-34950b34a161)

**Comment by huiye123-Hub** [comment]
欢迎<img src="https://img-static.mihoyo.com/communityweb/upload/ee7bb800b0c90f66fa9e321cb6d8ee17.png" alt="抱大腿" width="75" >

**Comment by 00zzxx** [comment]
你好哇<img src="https://bbs-static.miyoushe.com/static/2023/10/07/6feb23fa0de115295c11c3af4ab9a3b0_7714471584071518008.png" alt="卡琳_盯" width="75" >
